packages/shared/utils/filters.ts

```diff
@@ -38,17 +38,17 @@ export const bypassFilter: EventFilter = (invoke) => {
  *
  * @param ms
  */
-export function debounceFilter(ms: number) {
-  if (ms <= 0)
-    return bypassFilter
-
+export function debounceFilter(ms: MaybeRef<number>) {
   let timer: ReturnType<typeof setTimeout> | undefined
 
-  const filter: EventFilter = (invoke) => {
+  const filter: EventFilter = (invoke, args) => {
+    if (unref(ms) <= 0)
+      return bypassFilter(invoke, args)
```

Comment:
```suggestion
      return invoke()
```

Just like this will do, but after clearTimeout.

Reply:
😞 Sorry I tried that originally then typescript got mad. Had to reload VSCode to fix the error :/